EapWebInterface/src/main/java/de/hpi/unicorn/application/pages/input/generator/GeneratePanel.java

```diff
@@ -201,9 +266,9 @@ protected void onUpdate(AjaxRequestTarget target) {
         layoutForm.add(listContainer);
         listContainer.setOutputMarkupId(true);
 
-        DropDownChoice<EapEventType> dropDown = new DropDownChoice<>("eventTypeField",
+        eventTypeDropDown = new DropDownChoice<EapEventType>("eventTypeField",
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') Replace the type specification in this constructor call with the diamond operator ("<>"). [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://bpt-lab.org/sonarqube/coding_rules#rule_key=squid%3AS2293)